src/pymovements/dataset/dataset_download.py

```diff
@@ -218,6 +223,7 @@ def extract_dataset(
         paths: DatasetPaths,
         remove_finished: bool = False,
```

Comment:
can you add a `*,` before `remove_finished`? This is PR a breaking change either way, so we can make it more future proof.